source/opt/instruction.cpp

```diff
@@ -158,6 +158,10 @@ Instruction* Instruction::Clone(IRContext* c) const {
   clone->unique_id_ = c->TakeNextUniqueId();
   clone->operands_ = operands_;
   clone->dbg_line_insts_ = dbg_line_insts_;
+  for (auto& i : clone->dbg_line_insts_) {
+    i.unique_id_ = c->TakeNextUniqueId();
+    if (i.IsDebugLineInst()) i.SetResultId(c->TakeNextId());
+  }
```

Comment:
Not sure what you mean by "ok". I think these lines are necessary and correct because we have created new instances of Instruction which eventually need different result and unique ids.